central/pruning/pruning.go

```diff
@@ -507,6 +509,16 @@ func (g *garbageCollectorImpl) removeOrphanedPLOPs() {
 		log.Errorf("error removing PLOPs with no matching process indicator or process information: %v", err)
 	}
 	log.Infof("[PLOP pruning] Pruning of %d orphaned PLOPs with no matching process indicator or process information complete", prunedCount)
+
+	// Only run once since we don't expect any new PLOPs without poduids.
+	if !prunedPLOPsWithoutPodUIDs {
```

Comment:
This one looks more confusing than useful. It's going to be reset on every restart, yet making it look like there will be "only one" such pruning. If it has succeeded once, I assume subsequent calls are going to be essentially no-op, queries will delete nothing -- wouldn't it be lightweight enough to not bother?

Reply:
I think it's the same for other pruners, if they don't have anything to remove. So I'm +0.1c inclined towards removing this flag, unless you can show that it really can save something.

Reply:
I have removed the flag.